src/iris/api.py

```diff
@@ -1350,6 +1366,7 @@ def on_get(self, req, resp):
             }
 
         '''
+        check_param_list_len(req.params)
```

Comment:
Could this be called from one of the middleware functions common to all routes, rather than copy pasted into each route handler?

Reply:
yeah, I think it is used widely enough, and has minimal enough impact for endpoints it doesn't apply to that it makes sense, changed